### DISCUSSION TITLE
Możliwość ustawienia sortowania listy dyskusji po polu lastMessageCreationDate

### DISCUSSION THREAD

**Discussion by tomasz-grzesik** [allegro/allegro-api#8349] [Suggestions - rejected]
Od dziś dyskusje są sortowane po nowym polu **openedDate**, co niestety nic nie zmienia w temacie przeglądania dyskusji pod kątem nieodczytanych wiadomości.
Proponuję dodanie pola **sort** przy metodzie zwracającej listę dyskusji z możliwością przestawienia domyślnego pola **openedDate** na **lastMessageCreationDate**.

**Comment by PrzemyslawLukanowski** [comment]
Odrzuciliśmy sugestię, ponieważ w nowej wersji beta zasobu GET /messaging/threads, w którym zwracamy Problemy z zakupem, dodaliśmy parametr **read**, dzięki któremu możliwe jest filtrowanie wiadomości pod kątem tego, czy wątek został odczytany.
